packages/react/src/components/form/form-customisation.test.tsx

```diff
@@ -0,0 +1,248 @@
+import { render, screen } from "@testing-library/react";
+import { Form } from ".";
+import { TestSlashIDProvider } from "../../context/test-slash-id-provider";
+import { createTestUser, inputEmail } from "../test-utils";
+import { Slot } from "../slot";
+import userEvent from "@testing-library/user-event";
+import { ConfigurationProvider } from "../../main";
+import { useState } from "react";
+import { Factor } from "@slashid/slashid";
+import { Handle } from "../../domain/types";
+
+describe("#Form - customisation", () => {
+  test("should render the footer slot", () => {
```

Comment:
As per README.md:
 > In addition to that, any components passed in without a wrapper will be ignored in the top level containers.

I'd add a small test case for that

Reply:
Agreed, will add immediately!

Reply:
Done!